### ISSUE TITLE
Testing needed - HTTP server experiment/improvement

### ISSUE THREAD

**Issue by openshwprojects** [openshwprojects/OpenBK7231T_App#173]
I had some strange reports saying that in rare case on CB2S HTTP server does not respond. Some time ago I finally experienced the same issue on the WB3S. It was happening only on one of several WB3S that I have, seemingly chosen at random. I am not able to determine the root cause yet, but I have debugged it for some time and realized that only packets longer than about 1kb breaks stuff. Http was able to respond "/testmsg" query, because it's short, but it was failing on index, etc, because of long HTML style.
(to be more specific - if HTTP server was to send more than 1kb at once, it was getting stuck and thread was not proceeding any more, thus alloced buffers for http remained in memory and available memory was shrinking....)

**TLDR:** Can anyone (preferrably with UART flasher ready) check https://github.com/openshwprojects/OpenBK7231T_App/releases/tag/1.12.41 **and report to me if it works?**
It should load HTTP pages a bit slower, but should work 100% ok (and even more often than previous version).

PS: it also allocs only 1kb instead of 10kb in http client thread
